apps/bfd-db-migrator/pom.xml

```diff
@@ -111,12 +111,12 @@
 		<dependency>
 			<groupId>com.newrelic.telemetry</groupId>
 			<artifactId>dropwizard-metrics-newrelic</artifactId>
-			<version>0.5.0</version>
+			<version>0.7.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.newrelic.telemetry</groupId>
 			<artifactId>telemetry-http-okhttp</artifactId>
-			<version>0.6.1</version>
+			<version>0.13.2</version>
```

Comment:
Could we exercise the campsite rule and hoist the versions for `dropwizard-metrics-newrelic` and `telemetry-http-okhttp`  into the bfd-parent's pom.xml?